src/Symfony/Component/Security/Tests/Core/Encoder/BCryptPasswordEncoderTest.php

```diff
@@ -45,9 +45,6 @@ public function testCostInRange()
         }
     }
 
-    /**
-     * @requires PHP 5.3.7
```

Comment:
then use the password_compat check function to check for support (btw, the library used to trigger an error at runtime, which is probably why the skipping was there)